posts/decentralized-data-storage-finance.md

```diff
@@ -0,0 +1,51 @@
+
+## Solid Platform
+The Solid Platform was started by Tim Berners-Lee, a founder of the modern web, in 2015.  Berners-Lee's and team began updating the W3C specifications in 2018 to include the use of a distributed personal data store ([source: wikipedia.org](https://en.wikipedia.org/wiki/Solid_(web_decentralization_project))).  The Solid Project calls these distributed data stores 'pods'.  "Pods are like secure personal web servers for data. When data is stored in someone's Pod, they control which people and applications can access it." ([source: SolidProject.org](https://solidproject.org/)   In short, the Solid Specification allows the owner of data to make specific allowances for other organizations to access that data. For instance, [a physician could ask if you want to share your medical history](https://www.rsa.com/en-us/blog/2021-07/verifiable-credentials-the-key-to-trust-on-the-next-web) or a bank would ask if you wanted to share your tax returns for a loan application.
+
+
+## Verified Credentials
+
+The W3C Verified Credentials works with the Solid Platform to provide a decentralized layer of trust.  This is how you know that physician is who they say they are and the physician knows they are viewing and updating the right patient record.  Verified Credentials focuses on the storage and retrieval of 'claims' about identity that enable 'authorization'.  In the decentralized web, all this would still be stored on decentralized user controlled pods. For instance, a claim could be added by the government which includes a credential to drive.  This would authorize you to drive a car.  This decentralized Verified Credentials approach has attracted the attention of large groups such as OWSAP, RSA, and the UK NHS as a solution to the growing problem of data breaches.
+
+
+## Use Cases
+To date, distributed stores have seen a few use cases implemented within finance. The primary use cases implemented thus far fall into two categories.  1) customer convenience and 2) personalized customer experience.  Both these categories of use case are enabled by the concept of interoperability.  [explain concept]. Presumably this same interoperability is what would enable what  And the same interoperability that will enable "(improving their [the customer] experiences access the digital context)[https://canada-ca.github.io/PCTF-CCP/docs/RelatedPolicies/WEF_Digital_Identity.pdf]" as described in a World Economic Forum paper from Janurary 2020. 
```

Comment:
"Presumably this same interoperability is what would enable what" <- This sentence is incomplete

Reply:
fixed